src/Canvas.tsx

```diff
@@ -155,6 +158,34 @@ function Canvas<R, K extends keyof R>({
     selectCell({ rowIdx, idx }, true);
   }
 
+  useEffect(() => {
```

Comment:
Any particular reason to move this from DataGrid to Canvas?

Reply:
Need access to `eventBus` which is initialized in the Canvas component